src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,71 @@
+package codeu.chat.util;
+
+import java.io.IOException;
+import java.util.logging.Level;
+
+public final class Tokenizer {
+  
+  private StringBuilder token;
+  private String source;
+  private int at;
```

Comment:
I'd rename this variable to something a little more clear, like characterPosition